packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -62,7 +65,9 @@ class Layout extends React.Component {
     });
   }
   render() {
-    const { nav, helpItems, logo, siteAlert, intl, ...customProps } = this.props;
+    const {
+      nav, helpItems, logo, siteAlert, intl, ...customProps
+    } = this.props;
```

Comment:
doesn't linter like the concise one-line code? lol

Reply:
That's from running `eslint --fix` 🤷‍♂️

Reply:
🤷‍♂️ 🤷‍♂️ 🤷‍♂️

Reply:
`eslint --fix` will not necessarily align with the linter setting specified. Its generally recommended to not run it.